internals/generators/organization/config.yml.hbs

```diff
@@ -518,6 +518,13 @@ client:
       additional_fields: []
       logout_by_session: {{logout_by_session_ID}}
 
+  password_eye_icon:
```

Comment:
Just to make you understand why reworking the conf will be important I'm leaving this comment:
now that I tested this branch on a config that was created before this, the piece of code which calls this fails because it doesn't find it.

There's not action to do here, it's good enough for now, we'll improve these aspects in https://github.com/openwisp/openwisp-wifi-login-pages/issues/236 and https://github.com/openwisp/openwisp-wifi-login-pages/issues/235.